plugins/linguist-backend/src/api/LinguistBackendApi.ts

```diff
@@ -117,8 +126,11 @@ export class LinguistBackendApi {
       0,
       this.batchSize ?? 20,
     );
+    const { token } = await this.tokenManager.getToken();
```

Comment:
Theoretically, you're meant to call getToken for every single request, ie inside the forEach. It's meant to be caching and fast, so there's less risk for sudden expiry in the midst of ongoing work if it takes some time to finish.

Reply:
Done!